Problem 2/problem1Js.js

```diff
@@ -0,0 +1,18 @@
+(function (){
+    let integer = prompt("Enter an integer");
+    let array = [];
+    for(let i = 0; i < integer; i++){
+        array.push(Math.floor(Math.random() * 20 + 1));
+    }
+    array.sort(function(a, b){return a - b});
+    alert(array);
+    
+    let distnums = [];
```

Comment:
Good job in implementing your own way to find distinct count.
For some cases, using an appropriate data structure saves a lot of manual work and implementation.
So, I would suggest you to explore more javascript data structures.
For this example, using the Set data structure would have been better.